sdks/go/pkg/beam/core/runtime/graphx/serialize.go

```diff
@@ -195,7 +195,7 @@ func encodeFn(u *graph.Fn) (*v1.Fn, error) {
 		gen := reflectx.FunctionName(u.DynFn.Gen)
 		t, err := encodeType(u.DynFn.T)
 		if err != nil {
-			return nil, fmt.Errorf("bad function type: %v", err)
+			return nil, fmt.Errorf("failed to encode function %v, bad function type: %v", u, err)
```

Comment:
("failed to encode dynamic DoFn %v, bad function type: %v", u, err)

Reply:
Done.